merlion_perception/nodes/detect_bucket.py

```diff
@@ -40,8 +40,8 @@ def __init__(self, nodename, drive=None):
         self.init_markers()
 
         #sub to front cam
-        rospy.Subscriber("/logi_c920/usb_cam_node/image_raw", Image, self.img_callback, queue_size = 1)
-        rospy.Subscriber("/front/image_rect_color", Image, self.img_callback, queue_size = 1)
+        rospy.Subscriber("/logi_c920/usb_cam_node/image_raw", Image, self.front_img_callback, queue_size = 1)
```

Comment:
You can remove this line since I won't publish "/logi_c920/usb_cam_node/image_raw" anymore.